slash/fun.py

```diff
@@ -33,6 +33,19 @@ async def headsortails(self, interaction: Interaction):
         embed = nextcord.Embed(color=0x0DD91A, title=f"It is {hot}!")
         await interaction.response.send_message(embed=embed)
 
+    @slash(guild_ids=SLASH_GUILDS)
+    async def f(
+        self,
+        interaction: Interaction,
+        text: str = SlashOption(description="what you respect", required=False),
```

Comment:
Please capitalize the "What"
```suggestion
        text: str = SlashOption(description="What you respect", required=False),
```